src/main/java/thePackmaster/cards/serpentinepack/BrutalStrikes.java

```diff
@@ -35,48 +32,25 @@ public void use(AbstractPlayer abstractPlayer, AbstractMonster abstractMonster)
         addToBot(new DamageAction(abstractMonster, new DamageInfo(abstractPlayer, damage, DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.SLASH_DIAGONAL));
     }
 
-
-    public void applyPowers() {
-        AbstractPower vigor = AbstractDungeon.player.getPower(VigorPower.POWER_ID);
-        AbstractPower dice = AbstractDungeon.player.getPower(DicePower.POWER_ID);
-        if (vigor != null) {
-            vigor.amount *= this.magicNumber;
-        }
-        if (dice != null) {
-            dice.amount *= this.magicNumber;
-        }
-
-        super.applyPowers();
-
-        if (vigor != null) {
-            vigor.amount /= this.magicNumber;
-        }
-        if (dice != null) {
-            dice.amount /= this.magicNumber;
-        }
-    }
-
     @Override
     public void calculateCardDamage(AbstractMonster mo) {
-        AbstractPower vigor = AbstractDungeon.player.getPower(VigorPower.POWER_ID);
-        AbstractPower dice = AbstractDungeon.player.getPower(DicePower.POWER_ID);
-        if (vigor != null) {
-            vigor.amount *= this.magicNumber;
-        }
-        if (dice != null) {
-            dice.amount *= this.magicNumber;
-        }
-
-        super.calculateCardDamage(mo);
-
-        if (vigor != null) {
-            vigor.amount /= this.magicNumber;
-        }
-        if (dice != null) {
-            dice.amount /= this.magicNumber;
+        int debuffs = 0;
+        for (AbstractPower p : mo.powers){
+            if (p.type.equals(AbstractPower.PowerType.DEBUFF)){
+                debuffs++;
+            }
+        }
+        if (debuffs != 0){
```

Comment:
You don't really need the if/else here, since when `debuffs` is 0, all these calculations will result in nothing being changed. What you have will work correctly, just has some extra unnecessary code.

Reply:
technically results in more unnecessary operations, but ultimately it doesn't matter one way or another